### PR TITLE
Add Aditya Narayan to README.md

### PR DESCRIPTION
**Pull request by narayanaditya95**

### CODE REVIEW

**Comment by narayanaditya95** [comment]
@hargup Is the PR right?

File: README.md [hunk 1]
```diff
@@ -7,4 +7,5 @@ a new brach and add your name and send a pull request. Your entry should
 maintain the alphabetic order and it should be in the format `your name <email
 id>`.
 
-Harsh Gupta <gupta.harsh96@gmail.com>
+Harsh Gupta <gupta.harsh96@gmail.com>  
+Aditya Narayan <narayanaditya95@gmail.com>
```
Comment (abinashmeher999) [inline README.md]: @narayanaditya95 The names need to be in alphabetical order.

**Comment by abinashmeher999** [comment]
@narayanaditya95 squash both the commits into one.

**Comment by narayanaditya95** [comment]
@hargup @abinashmeher999 Rebased the commits into one and corrected the ordering. Please review.

**Comment by abinashmeher999** [comment]
:+1: from my side.

**Comment by hargup** [comment]
Looks Good :+1:

File: README.md [hunk 2]
```diff
@@ -7,4 +7,5 @@ a new brach and add your name and send a pull request. Your entry should
 maintain the alphabetic order and it should be in the format `your name <email
 id>`.
 
+Aditya Narayan <narayanaditya95@gmail.com>
```
Comment (hargup) [inline README.md]: Oops, I was sloppy and missed the trailing whitespaces.

Comment (narayanaditya95) [inline README.md]: Two trailing spaces to initiate a line break. Do I use the break tag <br /> instead?

Comment (hargup) [inline README.md]: Use a new line instead.

**Pull request merged**